functions/exercise4/script.js

```diff
@@ -1 +1,11 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+function validateAsPhraseConsistsOfLettersOnly(phrase) {
+    var validate = /^[a-zA-Z]*$/g.test(phrase);
+    if (validate) {
+        return true;
```

Comment:
Do you need this if? Why can't you simply return true after if where you validate if exception is needed?